spec/features/general_feedback_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+RSpec.feature 'Giving general feedback for the service' do
+  before(:each) { visit new_feedback_path }
+
+  context 'when viewing the visiting purpose section' do
+    it 'displays the section heading' do
+      expect(page).to have_content(I18n.t('general_feedback.visit_purpose_legend'))
```

Comment:
How much value does having these tests which assert against page content bring?

Reply:
I feel like asserting against content gives me more confidence that on the desired page (i.e that the test setup is correct). 
Although it might be enough to just assert on the name of the page.